ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -80,34 +80,65 @@
       }
     },
     {
+      "name": "verify ROWPARTITION is not queryable with feature disabled",
```

Comment:
I don't think this test and the one below are necessary. This behavior is good to check while developing the feature flag but doesn't feel necessary to check in for posterity. They will be removed once the feature flag is removed anyway.